pom.xml

```diff
@@ -139,7 +139,7 @@
         <servlet-api.version>6.0.0</servlet-api.version>
         <grizzly.version>4.0.0.payara-p1</grizzly.version>
         <jax-rs-api.impl.version>3.1.0</jax-rs-api.impl.version>
-        <jersey.version>3.1.0-M8</jersey.version>
+        <jersey.version>3.1.0</jersey.version>
```

Comment:
Does FISH-6435 not need reapplying?

Reply:
When testing I saw that we need to patch jersey version 3.1.0-M8 and add the dynamic proxy changes